### ISSUE TITLE
Pragmas gets unset on init

### ISSUE THREAD

**Issue by codito** [coleifer/peewee#1481]
This scenario uses runtime database configuration. The `pragmas` configuration is reset to empty upon doing a `SqliteDatabase.init(name)`. It's happening since `init` now takes `pragmas` as an input argument which defaults to `None`. It's a behavior change from peewee 2. Is it by design?

### Peewee 3.0.13
```python
Python 3.6.4 (default, Jan  5 2018, 02:35:40)                                                                                                                  
[GCC 7.2.1 20171224] on linux                                                                                                                                  
Type "help", "copyright", "credits" or "license" for more information.
>>> from peewee import SqliteDatabase                                                                                                                          
>>> db = SqliteDatabase(None, pragmas=(('foreign_keys', 'on'),))                                                                                                    
>>> print(db._pragmas)                                                                                                                                         
(('foreign_keys', 'on'),)                                                                                                                                      
>>> db.init(':memory:')                                                                                                                                        
>>> print(db._pragmas)                                                                                                                                         
()                                                                                                                                                             
```

### Peewee 2.10.2
```python
Python 3.6.4 (default, Jan  5 2018, 02:35:40)                                                                                                                  
[GCC 7.2.1 20171224] on linux                                                                                                                                  
Type "help", "copyright", "credits" or "license" for more information.
>>> from peewee import SqliteDatabase      
>>> db = SqliteDatabase(None, pragmas=(('foreign_keys', 'on'),))
>>> print(db._pragmas)
(('foreign_keys', 'on'),)
>>> db.init(':memory:')
>>> print(db._pragmas)
(('foreign_keys', 'on'),)
>>> 
```

**Comment by coleifer** [comment]
Thanks for reporting.

**Comment by coleifer** [comment]
Fixed, pushed new version to PyPI - 3.0.15. Thanks again for letting me know.

**Comment by codito** [comment]
Thanks a ton! Confirmed the issue is fixed with 3.0.15.
